go/kbfs/libfs/fs.go

```diff
@@ -691,7 +693,7 @@ func (fs *FS) Rename(oldpath, newpath string) (err error) {
 		err = translateErr(err)
 	}()
 
-	if err := fs.requireNonEmpty(); err != nil {
+	if err := fs.checkEmpty(false); err != nil {
```

Comment:
Can you make this an enum or make these calls include a comment as to what this boolean parameter means (e.g. `checkEmpty(true /*notExistErrIfEmpty*/)`